packages/modules/vehicles/mercedeseq/auth.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import sys
+import time
+import html
+import paho.mqtt.subscribe as subscribe
+import paho.mqtt.publish as publish
+sys.path.append("../../../")
+
+# call parameters
+EVId = str(sys.argv[1])
```

Comment:
```suggestion
ev_id = str(sys.argv[1])
```
Bitte auch die Python-Naming-Konventionen beachten. Klassen-Namen beginnen mit Großbuchstaben und nutzen CamelCase, Variablen Kleinbuchstaben und Unterstrich.

Reply:
done